getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Getting started with VPP networking
+description: Install Calico with the VPP dataplane on a Kubernetes cluster
+canonical_url: '/getting-started/kubernetes/vpp/getting-started'
+---
+
+
+### Big picture
+
+Install {{site.prodname}} and enable the tech preview of the new VPP dataplane.
+
+> **Warning!** The VPP dataplane is a tech preview and should not be used in production clusters. It has had limited testing and it will contain bugs (please report these on the [Calico Users slack](https://calicousers.slack.com/archives/C017220EXU1) or [Github](https://github.com/projectcalico/vpp-dataplane/issues)).  In addition, it does not support all the features of {{site.prodname}} and it is missing some security features such as Host Endpoint policies.
+{: .alert .alert-danger }
+
+### Value
+
+The new VPP dataplane mode has several advantages over standard linux networking pipeline mode:
```

Comment:
The {{site.prodname}} VPP dataplane mode has several advantages over standard Linux networking pipeline mode:

* Scales to higher throughput, especially with WireGuard encryption enabled
* Improves IPsec encryption performance
* Native support for Kubernetes services without needing kube-proxy, which:
  * Reduces first-packet latency for packets to services
  * Preserves external client source IP addresses all the way to the pod

The VPP dataplane is entirely compatible with the other {{site.prodname}} dataplanes, meaning you can have a cluster with VPP-enabled nodes along with regular nodes. This makes it easy to migrate a cluster from Linux or eBPF Calico networking to VPP networking.

Trying out the tech preview will give you a taste of these benefits and an opportunity to give feedback to the Calico/VPP team.

In the future, the VPP dataplane will offer additional features for network-intensive applications, such as providing `memif` userspace packet interfaces to the pods (instead of regular Linux network devices).